README.md

```diff
@@ -30,26 +30,17 @@ for example code.
 
 ## Local Development
 
-### `npm start`
+### `npm run start -- BACKEND NETWORK CONFIG-DIR STATE-DIR`
```

Comment:
This is not the same as before. Previously it just watched the sources and rebuilt `index.ts` but didn't run anything, which was convenient for me, and is what devs would want I think.

Reply:
`start` in my mind implies the program is running, which in our case is the CLI. The behaviour described is a build watch: `npm run build -- --watch`

Reply:
BTW, this is where `yarn` has an advantage with it's script execution: `yarn build --watch`

Reply:
@rvl are you happy with this?

Reply:
Not really happy - because I doubt anyone would want the the cli program to run every time a file changes.

But now you mention `npm run build -- --watch`. This is exactly what I want. This command should be added to the readme and the start script removed.

I don't understand what the advantage of adding yarn would be here.

Reply:
I'm not advocating to switch to yarn, just pointing out there's a simpler script interface